.github/workflows/ci.yml

```
@@ -18,18 +18,21 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: "1.16"
+      - uses: pnpm/action-setup@v2.0.1
```

Better to pin just the major version so we get any bug fixes or performance improvements that appear in a minor or patch release.
```suggestion
      - uses: pnpm/action-setup@v2
```

---

Ok sounds good.